spec/systems/keywords/view_keyword_detail_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'View keyword details', type: :system do
+  include ActiveJob::TestHelper
+
+  # rubocop:disable RSpec/ExampleLength
+  it 'displays a link to keywords#show for succeeded keyword and displays the pagination nav' do
```

Comment:
The test does not describe what it does, as there are many assertions for the scraped content. The link to the homepage is a minor detail.

```suggestion
  it 'displays the scraped content information' do
```

Reply:
Fixed in 9be9a361e317cafab5bb7ab0eb04e3b714f1581f